exp-models/tests/unit/utils/csv-writer-test.js

```diff
@@ -120,16 +120,44 @@ test('uniqueFields', function(assert) {
 test('writeCSV', function(assert) {
     const data = [
         {
-            test: one
+            test1: one,
+            test2: one
         },
         {
-            test: two
+            test1: two,
+            test2: two
         }
     ];
 
     assert.strictEqual(
-        writeCSV(data, ['test']),
-        `test${NL}"one"${NL}"two"`
+        writeCSV(data, ['test1', 'test2']),
+        [
+            'test1,test2',
+            '"one","one"',
+            '"two","two"'
+        ].join(NL)
+    );
+});
+
+test('writeCSV with tabs', function(assert) {
+    const data = [
+        {
+            test1: one,
+            test2: one
+        },
+        {
+            test1: two,
+            test2: two
+        }
+    ];
+
+    assert.strictEqual(
+        writeCSV(data, ['test1', 'test2'], '\t'),
```

Comment:
Good to verify that the separator argument works!

For the next step (adding to experimenter), we're going to do away with the TSV output option altogether and replace it with CSVs; it was really just there until we had the code to write CSV files correctly.

(The only difference is the delimiter; using tab-delimited let us build a quick prototype without worrying about edge cases. CSV is more familiar to researchers though)